### PR TITLE
homework 3

### PR DESCRIPTION
**Pull request by codewhynot**

### CODE REVIEW

File: js-core/homeworks/homework-3/src/main.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    и возвращать новый объект изменяя свойство name
+    в большой регистр
+    */
+    let myObj = {
+        name: 'Some object name'
+    }
+
+    function returnObj (obj){
+        let modifyObj = {
+            name: String(obj.name).toUpperCase()
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-3/src/main.js]: Not sure we have to use *String* here, it's a bit overhead. 

I will talk about it in classrom

File: js-core/homeworks/homework-3/src/main.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    -> @@ SUPER !
+    Напишите функцию, которая будет
+    возвращать 'Fizz' если передаваемый параметр кратен 3,
+    'Buzz', если передаваемый параметр кратен 5,
+    'FizzBuzz' - если параметер кратен 3 и 5
+    Если передаваемое число не кратно 3 или 5, то вернуть указанный параметр
+    */
+
+    function fizzBuzz(param){
+        if((param % 3) == 0 && (param % 5) == 0){return 'FizzBuzz'}
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-3/src/main.js]: There some styles changes required, please don't make a single line with if statement.

Furthermore, some conditionals can be simplified, let we think can we improve it?

File: js-core/homeworks/homework-3/src/main.js [hunk 3]
```diff
@@ -120,11 +120,9 @@
     Если передаваемое число не кратно 3 или 5, то вернуть указанный параметр
     */
 
-    function fizzBuzz(param){
-        for (var i = 1; i <= param; i++) {
-            var f = param % 3 == 0, b = param % 5 == 0;
-            return f ? b ? "FizzBuzz" : "Fizz" : b ? "Buzz" : i
-          }
+    function fizzBuzz(param){   
+        var f = param % 3 == 0, b = param % 5 == 0;
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-3/src/main.js]: :))) we are not going to make it unreadable, it's bad practice.

Ok I leave it as-is, and going to merge in friday :)

**Review by OlegLustenko — APPROVED** [review]

**Comment by codewhynot** [comment]
Можно замерджить эту ветку? )))

**Review by OlegLustenko — APPROVED** [review]

**Pull request merged**